ledger/sandbox-on-x/src/test/lib/scala/com/daml/ledger/sandbox/MainWithEphemeralOracleUser.scala

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.sandbox
+
+import com.daml.ledger.resources.ResourceContext
+import com.daml.ledger.sandbox.SandboxOnXRunner
+import com.daml.resources.ProgramResource
+import com.daml.testing.oracle.OracleAround
+
+object MainWithEphemeralOracleUser {
+  def main(args: Array[String]): Unit = {
+    val user = OracleAround.createNewUniqueRandomUser()
```

Comment:
Can't we have `OracleAround` as a trait?

Reply:
Why?

Reply:
Because we have a `PostgresAround` trait. Just for similarity

Reply:
I did, since I rewrote it as an Object from a trait in my previously merged PR. ;) I am happy to receive any feedback, but can you please do that on that PR? https://github.com/digital-asset/daml/pull/12573
(except if being as an object is causing some problems here of course)

Reply:
Ok, fine with leaving it as is